src/main/java/com/algorand/algosdk/v2/client/Utils.java

```diff
@@ -1,19 +1,162 @@
 package com.algorand.algosdk.v2.client;
 
+import java.math.BigInteger;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+
+import com.algorand.algosdk.crypto.Address;
+import com.algorand.algosdk.transaction.SignedTransaction;
+import com.algorand.algosdk.transaction.Transaction;
+import com.algorand.algosdk.transaction.Transaction.Type;
 import com.algorand.algosdk.v2.client.common.AlgodClient;
 import com.algorand.algosdk.v2.client.common.Response;
+import com.algorand.algosdk.v2.client.model.DryrunRequest;
+import com.algorand.algosdk.v2.client.model.Application;
+import com.algorand.algosdk.v2.client.model.ApplicationParams;
+import com.algorand.algosdk.v2.client.model.ApplicationStateSchema;
+import com.algorand.algosdk.v2.client.model.Account;
+import com.algorand.algosdk.v2.client.model.Asset;
 import com.algorand.algosdk.v2.client.model.NodeStatusResponse;
 import com.algorand.algosdk.v2.client.model.PendingTransactionResponse;
 
 public class Utils {
+
+    // https://github.com/algorand/go-algorand/blob/e466aa18d4d963868d6d15279b1c881977fa603f/libgoal/libgoal.go#L1089-L1090
+    public static Long defaultAppId = 1380011588L;
+
+    /**
+     * Wait until a transaction has been confirmed or rejected by the network
+     * or wait until waitRound fully elapsed
```

Comment:
Wrong description.

It might be good to go into a lot of detail about how this function is used since it's a standalone feature-function that does a lot of stuff under the hood. If you want to go crazy, you could try embedding a code snippet ([with javadoc syntax](https://stackoverflow.com/a/542142)). I don't think we do that anywhere else though, so consider it optional.